BitwardenShared/UI/Platform/Application/AppCoordinator.swift

```diff
@@ -389,4 +417,4 @@ extension AppCoordinator: VaultCoordinatorDelegate {
     func presentLoginRequest(_ loginRequest: LoginRequest) {
         showLoginRequest(loginRequest)
     }
-}
+} // swiftlint:disable:this file_length
```

Comment:
⛏️ Don't we disable at the tops of files and enable at the bottoms of files?

Reply:
Not sure, I've seen this in a couple of places already, but I can change it if needed.